### PR TITLE
Also update the kernel command line arguments in the GRUB environment file

### PR DESCRIPTION
**Pull request by sloanebernstein**
Kernels installed after ELevate has upgraded the system do not hard-code their command line arguments into the individual bootloader entry but instead refer to a variable in GRUB's persistent environment, stored in `/boot/grub2/grubenv`. Although a full `grub2-mkconfig` does appear to update the `kernelopts` GRUB environment variable from `GRUB_CMDLINE_LINUX` in the contents of `/etc/default/grub`, the only time this is run when a kernel is installed, it specifically excludes environment variable updates. The only other thing that explicitly touches the GRUB environment is to change the `saved_entry` variable to update the default boot option.

Since a full `grub2-mkconfig` is overkill just to update the environment, use the `grub2-editenv` utility to update the appropriate variable.

By submitting pull requests to this repo, I agree to the Contributor License Agreement which can be found at: https://github.com/cpanel/elevate/blob/main/docs/cPanel-CLA.pdf

### CODE REVIEW

**Review by atoomic — APPROVED** [review]

File: elevate-cpanel [hunk 1]
```diff
@@ -1148,6 +1150,26 @@ sub check_and_fix_grub ($self) {
     return unless $grub_conf =~ s/GRUB_CMDLINE_LINUX="(.+?)"/GRUB_CMDLINE_LINUX="$1 net.ifnames=0"/m;
     File::Slurper::write_binary( DEFAULT_GRUB_FILE, $grub_conf );
 
+    # Also save net.ifnames=0 to the GRUB environment:
+    my $grubenv = Cpanel::SafeRun::Simple::saferunnoerror( GRUB_EDITENV, GRUB_ENV_FILE, 'list' );
+    foreach my $line ( split "\n", $grubenv ) {
+        my ( $name, $value ) = split "=", $line, 2;
+        next unless $name eq "kernelopts";
+
+        if ( $value !~ m/\bnet\.ifnames=0\b/a ) {
```
Comment (atoomic) [inline elevate-cpanel:1159]: probably better to check for `net.ifnames=` and not the value?

Comment (sloanebernstein) [reply elevate-cpanel:1159]: > probably better to check for `net.ifnames=` and not the value?

probably

Comment (toddr) [reply elevate-cpanel:1159]: looks good except for this.

Comment (toddr) [reply elevate-cpanel:1159]: Let's remove `0\b` from the regex and I think we're good.

**Pull request merged**